app/src/main/java/com/sun/music61/data/source/local/TracksLocalDataSource.java

```diff
@@ -0,0 +1,60 @@
+package com.sun.music61.data.source.local;
+
+import android.content.Context;
+import android.database.Cursor;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.google.common.base.Function;
+import com.sun.music61.data.model.Track;
+import com.sun.music61.data.source.TracksDataSource;
+
+import java.util.List;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Concrete implementation of a data source as a db.
+ */
+public class TracksLocalDataSource implements TracksDataSource {
+
+    @Nullable
+    private static TracksLocalDataSource INSTANCE;
```

Comment:
bạn check lại cho mình coding convention 
```suggestion
    private static TracksLocalDataSource sIntance;
```